webrtc.html

```diff
@@ -15620,23 +15617,6 @@ <h3>
                 {{RTCDataChannelStats/bytesReceived}}
               </td>
             </tr>
-            <tr>
-              <th class="row">
-                {{RTCStatsType/"sender"}}
-              </th>
-              <td rowspan="2">
-                {{RTCMediaHandlerStats}}
-              </td>
-              <td rowspan="2" data-tests=
-              "RTCPeerConnection-track-stats.https.html,RTCPeerConnection-mandatory-getStats.https.html">
-                {{RTCMediaHandlerStats/trackIdentifier}}
```

Comment:
Oh we should make it mandatory to implement RTCInboundRtpStreamStats.trackIdentifier. It got moved there.

Reply:
can you provide it as a separate PR since that one got merged now? :)

Reply:
Will do https://github.com/w3c/webrtc-pc/issues/2747

Reply:
Done: https://github.com/w3c/webrtc-pc/pull/2748